NPSpecies - species data for 15 most visited National Parks in the USA (#756)
rfordatascience/tidytuesday: data/curated/NPSpecies/intro.md

"The information in NPSpecies is available to the public. The exceptions to this are records for some sensitive, threatened, or endangered species, where widespread distribution of information could potentially put a species at risk.   
 
An essential component of NPSpecies is evidence; that is, observations, vouchers, or reports that document the presence of a species in a park. Ideally, every species in a park that is designated as “present in park” will have at least one form of credible evidence substantiating the designation"

If you are looking for more detailed information on the dataset, here is the glossary for column names, field options, and tag meanings: 
https://irma.nps.gov/content/npspecies/Help/docs/NPSpecies_User_Guide.pdf

To properly cite NPSpecies use the following: 
**NPSpecies - The National Park Service biodiversity database.**
**https://irma.nps.gov/npspecies/. Accessed date/time. **

*This data was accessed on September 2nd, 2024.*

> P.S. if interested in additional data, below is the curated dataset for all national parks:
> https://github.com/frankiethull/NPSpecies
> 
> 
> > Information on Species in National Parks
> 
> QUESTION?

Reviewer: ```suggestion
If you are interested in additional data, the curated dataset for all national parks is available at [https://github.com/frankiethull/NPSpecies](https://github.com/frankiethull/NPSpecies).
```
